regions/io/crtf/read.py

```diff
@@ -362,7 +363,7 @@ def convert_coordinates(self):
                 else:
                     self._raise_error("Not in proper format: {0} should be a pair of length".format(y))
             if x == 'l':
-                if isinstance(y, str):
+                if 'str' in str(type(y)) or 'unicode' in str(type(y)):
```

Comment:
I believe this can be better accomplished by doing something like:
```
from six import string_types
if isinstance(y, string_types):
    ...
```
see https://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility.  We might be able to import six from astropy, but that might also have been deprecated...